doc/elements-confidential-transactions.md

```diff
@@ -0,0 +1,480 @@
+see [pset.mediawiki](./pset.mediawiki) for more information.
+
+## Elements RPCs
+
+RPCs that are directly related to Confidential Transactions are listed here in the groups listed in the Elements help text. Note that some raw transaction RPCs appear in the Wallet section. See the Elements RPC help for details on parameters and invocation.
+
+### Util
+
+`createblindedaddress`
+Creates a confidential address by combining an unconfidential address
+and a public blinding key.
+
+### Raw Transactions
+
+`analyzepsbt`
+Provides information about the blinding status of a PSET’s outputs.
+
+`createpsbt`
+All blinding parameters may be set explicitly when defining a PSET.
+
+`decoderawtransaction`
+Displays a JSON representation of a raw transaction. Useful for
+inspecting confidential parameters. When the wallet is able to unblind
+a parameter, the unblinded value will replace the blinded value. The
+"ConfidentialNonce" parameter will be displayed as "commitmentnonce".
+
+`gettransaction`
+For a transaction in this wallet, displays properties of the
+transaction, including some confidential parameters such as asset and
+value blinders ("blinding factors").
+
+`getrawtransaction <txid> true`
+With `verbose=true` and `txindex=1`, `getrawtransaction` will display
+public details of a transaction’s confidential parameters. Use
+`unblindrawtransaction` followed by `decoderawtransaction` to unblind and
+decode the transaction’s private parameters.
+
+`blindrawtransaction`
+Blinds the outputs of a raw transaction (as might be created by
+`createrawtransaction`), using only wallet inputs.
+
+`rawblindrawtransaction`
+Blinds the outputs of a raw transaction (as might be created by
+`createrawtransaction`). This RPC requires all blinding factors be
+provided explicitly. 
+
+
+### Wallet  - Keys and Addresses
+
+`getnewaddress`
+By default, generates a confidential address encoded as blech32 (see
+"Confidential Addresses" section above). The public key is embedded in
+the address along with the ScriptPubKey. A confidential address is a
+tuple (confidential_key, unconfidential address). 
+
+`getaddressinfo`
+Displays the (public) confidential and unconfidential properties of an address.
+
+`dumpblindingkey`
+Reveals the private blinding key for a confidential address. A
+third-party will need this key to unblind transactions (see
+"Third-party Unblinding" below). 
+
+`dumpissuanceblindingkey`
+Reveals the private blinding key that was used to blind the amounts on
+an issuance input. This key is required when using reissuance tokens. 
+
+`dumpmasterblindingkey`
+Reveals the wallet’s master blinding key from which all blinding keys for generated addresses are derived.  See SLIP-007.
+
+`importaddress`
+A confidential address may be imported at any time. However, in order
+to unblind outputs for a confidential address, it is necessary to also
+import the blinding key for that address’ confidential_key. See
```

Comment:
What is a `confidential_key`? This is the first time I've seen the term mentioned.

Could this read:
```suggestion
import the blinding key for that address. See
```

Reply:
Update: after reading the whole document, I see that it is defined later. However, I still believe this simplification makes sense.